deployer/src/test/java/eu/seaclouds/apache/PhpHttpdServerIntegrationTest.java

```diff
@@ -49,7 +49,7 @@
 
     private TestApplication app;
     private String gitRepoURLApp="https://github.com/kiuby88/phpHelloWorld.git";
-    private String tarballResourceUrl="https://github.com/kiuby88/phpHelloWorld/archive/1.tar.gz";
+    private String tarballResourceUrl="https://s3-us-west-2.amazonaws.com/seaclouds-usescases-artifacts/php/phpHelloWorld.tgz";
```

Comment:
When the amazon account expires, this will not work anymore, I think. Could this tarball be placed inside src/test/resources?